### PR TITLE
Accessibility alignments

### PR DESCRIPTION
**Pull request by SergSkol**
Accessibility Report:
- Page titles - Add author's name, like "Sergiy's Portfolio" https://github.com/SergSkol/Portfolio/blob/0e367bb074f73674aeaf19e90a468efe349c5c51/index.html#L13
- Image text alternatives - Add alts for 5 images https://github.com/SergSkol/Portfolio/blob/0e367bb074f73674aeaf19e90a468efe349c5c51/index.html#L54
- Text headings - Change header to h2 for class work-card-title https://github.com/SergSkol/Portfolio/blob/0e367bb074f73674aeaf19e90a468efe349c5c51/index.html#L66
- Color contrast - no issues (following Figma design) 
- Resize - no issues
- Interaction - no issues
- Moving content - no issues
- Multimedia - no issues
- The basic structure of the page - Make changes meaned above

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -80,10 +82,10 @@ <h1 class="my-recent-works" id="portfolio">My Recent Works</h1>
...
         <div class="work-card">
-          <header class="work-card-title">
+          <h2 class="work-card-title">
             Multi-Post Stories<br>
             Gain+Glory
-          </header>
+          </h2>
           <ul class="work-card-tags nobull">
             <li class="work-card-tag">Ruby on rails</li>
             <li class="work-card-tag">css</li>
```
Comment (Reem-lab) [inline index.html:91]: - [ ] For the button for see project I think here you have low contrast errors so please make it normal you can use this extension to make it normal for all the button as you can see here  very low contrats between text and background image so kindly fix it, you can use this extension to check [wave](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)

image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻 


Good job so far! 🚀💪🏻💯🔥🔥🔥🔥
There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionalit✅
📌 Good commit messages ✅
📌 Professional README ✅

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -207,10 +212,12 @@ <h1 class="about-me-title">About me</h1>
         </ul>
       </div>
     </section>
+
+    <!-- CONTACTS -->
     <section class="contact-me" id="contact">
-      <p class="contact-me-text">I'm always interested in hearing about new projects,
+      <h1 class="contact-me-text">I'm always interested in hearing about new projects,
```
[the remark this answers is not in the document]
Comment (SergSkol) [reply index.html:218]: Thank you! Done!

File: index.html (hunk 1 above)
Comment (SergSkol) [reply index.html:91]: Thank you! Done!

**Review by Santiago220991 — APPROVED** [review]
Hi @SergSkol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
:heavy_check_mark: Applied changes suggested by the last code reviewer.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**